packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1045,6 +1062,29 @@ class _CupertinoDatePickerDateTimeState extends State<CupertinoDatePicker> {
     );
   }
 
+  // Builds the time seperator column.
+  Widget _buildTimeSeperatorWidget(
```

Comment:
Would this be better as a separate private StatelessWidget instead of just a builder method?

Reply:
I agree with you, I've created _buildTimeSeperatorWidget following other columns building methods hours, minutes